### PR TITLE
Add filter for shared users

### PR DESCRIPTION
**Pull request by IgorZiegler**

### CODE REVIEW

**Comment by IgorZiegler** [comment]
Related to https://github.com/systemapic/wu/issues/277

**Comment by knutole** [comment]
Same comment (https://github.com/systemapic/systemapic.js/issues/19) applies here.

**Comment by knutole** [comment]
Small bug when trying to Backspace (delete) in input field.. Should remove the user that is nearest cursor (in pitcure below, that's Igor). 

Make sure that added user is still added to `this._access` ... and reload page to check that user is actually added. ALSO - check that the user get's the access he's given. (Log in as the user to see if you have access to the shared project)

screen shot 2016-02-08 at 3 52 49 pm

File: js/src/chrome/data/chrome.data.js [hunk 1]
```diff
@@ -1479,6 +1479,27 @@ Wu.Chrome.Data = Wu.Chrome.extend({
 		var allUsers = _.sortBy(_.toArray(app.Users), function (u) {
 			return u.store.firstName;
 		});
+		var itemsContainers = [];
+		var checkedUsers = {};
+
+		function onKeyUp(e) {
```
Comment (knutole) [inline js/src/chrome/data/chrome.data.js]: not a big fan of functions inside functinos.. perhaps you could refactor to `this._onKeyUp` ?

**Comment by IgorZiegler** [comment]
@knutole We aren't use this._access field in that module. I think it is a copy paste from project. I will try to fix it.

**Comment by knutole** [comment]
@IgorZiegler yes, you might be right... :)

**Comment by knutole** [comment]
@IgorZiegler how did this go? fixed?

**Pull request merged**